pyt/formatters/text.py

```diff
@@ -0,0 +1,23 @@
+"""This formatter outputs the issues as plain text."""
+
+
+def report(
+	vulnerabilities,
+	fileobj
+):
+	"""
+	Prints issues in text format.
+
+	Args:
+		vulnerabilities: list of vulnerabilities to report
+		fileobj: The output file object, which may be sys.stdout
+	"""
+	number_of_vulnerabilities = len(vulnerabilities)
+	with fileobj:
+		if number_of_vulnerabilities == 1:
+			fileobj.write('%s vulnerability found:' % number_of_vulnerabilities)
+		else:
+			fileobj.write('%s vulnerabilities found:' % number_of_vulnerabilities)
```

Comment:
if there is no vulnerabilities on scanned file, the output being "0 vulnerability found:" i think if  we can set "elif number_of_vulnerabilities == 0" would be better

Reply:
So right now it'll print out "0 vulnerabilities found:", which makes more sense English-wise than "0 vulnerability found:", but it could maybe be improved by saying "No vulnerabilities found." Thoughts?

Reply:
"No vulnerabilities found."  it sounds better in the ear

Reply:
Yes it does, I also found I forgot to add `\n` when I tested it, thanks :)

Reply:
You are welcome

Reply:
Do you want to approve? You'll get a green square 😁

Reply:
It's ok for me :)